steps/appdome-build-2secure-android/1.0.1/step.yml

```diff
@@ -0,0 +1,52 @@
+title: Appdome Build
+summary: |
+  Builds a mobile app using Appdome's platform
+description: |
+  Integration that allows activating security and app protection features, bulding and signing mobile apps using Appdome's API.
+website: https://github.com/Appdome/bitrise_build-2secure-android
+source_code_url: https://github.com/Appdome/bitrise_build-2secure-android
+support_url: https://github.com/Appdome/bitrise_build-2secure-ios/issues
+published_at: 2023-04-10T11:50:05.373709+03:00
+source:
+  git: https://github.com/Appdome/bitrise-step-appdome-build-2secure-android.git
+  commit: 2a3d068aef3a8a4877e69f48ba7ca25515ea8ac2
+project_type_tags:
+- android
+type_tags:
+- build
+- code-sign
+toolkit:
+  bash:
+    entry_file: step.sh
+inputs:
+- app_location: null
```

Comment:
Does this input expect an apk or aab file? If yes then we have default Bitrise specific outputs which could be used as default values so users need to enter less things. 

We have `BITRISE_APK_PATH` and `BITRISE_AAB_PATH `.

Reply:
I don't know which kind of app (apk or aab) the user will be building, so I left it blank for him to fill it.